### PR TITLE
Add atoms aa1 to implement 1

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently when reloading or switching a page it starts not on top of the page. #14 
- Currently there is no project component. #15 
- Currently there is no component for pinned items. #16 

**What is the new behavior (if this is a feature change)?**
- Now the spawn point of a page is on top. #14 
- Now there is a project component which could display several repository data. #15 
- Now there is a component for pinned items. #16

**Other information**:
- 🐍

### CODE REVIEW

File: src/components/atoms/Project/index.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            </div>
+          </li>
+        </a>
+      </MDBCol>
+    );
+  }
+}
+//#endregion
+
+//#region > PropTypes
```
Comment (pinterid) [inline src/components/atoms/Project/index.jsx:57]: Is that a valid region?

File: src/components/atoms/ScrollToTop/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+//> React
```
Comment (pinterid) [inline src/components/atoms/ScrollToTop/index.jsx:2]: Missing import anchor.

File: src/components/atoms/ScrollToTop/index.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+// Contains all the functionality necessary to define React components
+import React from "react";
+// DOM bindings for React Router
+import { withRouter } from "react-router-dom";
+
+/**
```
Comment (pinterid) [inline src/components/atoms/ScrollToTop/index.jsx:10]: Missing component anchor.

File: src/components/atoms/ScrollToTop/index.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      // Scroll to top of viewport
+      window.scrollTo(0, 0);
+    }
+  }
+
+  render() {
+    return this.props.children;
+  }
+}
+
```
Comment (pinterid) [inline src/components/atoms/ScrollToTop/index.jsx:30]: Missing export anchor.

**Review by pinterid — CHANGES_REQUESTED** [review]
Some changes have to be done.

File: src/components/atoms/Project/index.jsx (hunk 1 above)
Comment (pinterid) [reply src/components/atoms/Project/index.jsx:57]: It is!

**Review by pinterid — APPROVED** [review]
Looks good to merge.

File: src/components/atoms/Project/index.jsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+// React PropTypes
+import PropTypes from "prop-types";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import { MDBCol, MDBIcon } from "mdbreact";
+//#endregion
+
+//#region > Components
+/**
+ * @class A project which contains repository data
```
Comment (kleberbaum) [inline src/components/atoms/Project/index.jsx]:
```suggestion
 * @class A component which contains repository data
```

File: src/components/atoms/Pinned/index.jsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+// "Material Design for Bootstrap" is a great UI design framework
+import { MDBRow, MDBCol, MDBCard, MDBCardBody, MDBIcon } from "mdbreact";
+
+//> CSS
+import "./pinned.scss";
+//#endregion
+
+//#region > Components
+/**
+ * @class Displays pinned projects
```
Comment (kleberbaum) [inline src/components/atoms/Pinned/index.jsx]:
```suggestion
 * @class A component which contains pinned projects
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**